error.go

```diff
@@ -44,6 +44,9 @@ var guestNamespaceCreationErrorSuffix = "namespaces/kube-system/serviceaccounts:
 // match example https://play.golang.org/p/ipBkwqlc4Td
 var guestDNSNotReadyPattern = "dial tcp: lookup .* on .*:53: no such host"
 
+// match example https://play.golang.org/p/iiYvBhPOg4f
```

Comment:
The godoc is not 100% because it should start with the thing it describes by convention.